### ISSUE TITLE
Allow disco in one project to monitor a set of other projects

### ISSUE THREAD

**Issue by steren** [mchmarny/disco#47]
It seems that disco deployed as a service can only discover resources from that project.

I would prefer to deploy disco in a dedicated project, and then give it read access on the project(s) I want to monitor.

This would allow me to not "pollute" my production projects and to easily uninstall it (by deleting the project)

**Comment by mchmarny** [comment]
Disco uses the access granted to the service account under which it runs to discover projects/services. That very use-case you described is supported by granting the SA from one project the necessary rights in the other project. 

Created new issue to track this #60

**Comment by mchmarny** [comment]
The docs for cross project disco service discovery have been updated.

https://github.com/mchmarny/disco/blob/main/docs/SERVICE.md#deployment

TLDR; granting the disco service account created by the deployment `Viewer` role on any other project will make the services in that project discoverable to disco.
